src/behavior.scm

```diff
@@ -203,6 +203,29 @@
 			(ListLink (Node "--- Glance at new person")))
 	))
 
+;; New recognized person sequence
+(DefineLink
+	(DefinedPredicate "Interacting Sequence for recognized person")
+	(SequentialAnd
+		;(True (DefinedSchema "glance at new person"))
+		(True (Put ; Setting interaction target for "look at person"
+			(DefinedPredicate "Set interaction target")
+			(RandomChoice (Put; FIXME Replace by multiple face tracking.
+				(DefinedSchemaNode "Get recognized face's face id")
+				(DefinedSchema "Get recognized faces")))))
+		(DefinedPredicate "look at person")
+		(True (Put
+			(DefinedPredicate "Greet recognized person")
```

Comment:
This is again a complete short-circuit of openpsi and of the language pipeline.

The flow should be: when a person is recognized, a flag should be set "we just now recognized person in the room".  Then, there is a separate openpsi rule that decides whether or not it wants to say hello to the person.

The flag for  "we just now recognized person in the room" could work in a very similar way to "there is a new person in the room (who we haven't acked)"  

I don't quite remember how it works, but I think ther is a psi rule that triggers when "there is a new person in the room (who we haven't acked)"  and if the rule triggers, then the robot turns in that direction.    Similarly, there should be a psi rule that triggers when someone is recognized, which then runs an AIML rule ("How's it hangin' <name>", "Que pasa, <name>", "Long time no see <name>" instead of hard-coded "Hello <name>") or, better yet, generates some concept nodes and pumps it through sureal.